aci/resource_aci_l3extvirtuallifp.go

```diff
@@ -246,6 +250,43 @@ func setVirtualLogicalInterfaceProfileAttributes(l3extVirtualLIfP *models.Virtua
 	return d, nil
 }
 
+func getL3extRsVSwitchEnhancedLagPol(client *client.Client, dn, tDn string) string {
+	l3extVirtualLIfPLagPolAttDn := fmt.Sprintf("%s/rsdynPathAtt-[%s]/vlifplagpolatt", dn, tDn)
```

Comment:
You use %s/rsdynPathAtt-[%s]/vlifplagpolatt 3 times, can we add it to the go client or somewhere to reuse it?